packages/electron-filestore/test/filestore.test.ts

```diff
@@ -148,6 +148,21 @@ describe('FileStore', () => {
       expect(dumps[1].minidumpPath).toEqual(join(base, 'report02.dmp'))
       expect(dumps[1].eventPath).toEqual(join(store.getPaths().runinfo, id))
     })
+
+    it('scans subdirectories', async () => {
+      const base = store.getPaths().minidumps
+
+      await writeFile(join(base, 'some-other-thing.ps'), 'not a crash')
+      await createMinidump('report01.dmp', '')
+      await createMinidump(join('reports', 'report02.dmp'), id)
+      await createMinidump(join('pending', 'report03.dmp'), id)
+
+      const dumps = await store.listMinidumps() as any[]
+      const dumpPaths = dumps.map(d => d.minidumpPath)
+      expect(dumpPaths).toContain(join(base, 'report01.dmp'))
+      expect(dumpPaths).toContain(join(base, 'reports', 'report02.dmp'))
+      expect(dumpPaths).toContain(join(base, 'pending', 'report03.dmp'))
```

Comment:
To be a bit stricter, is it possible to use `toEqual` here instead of checking each path individually? I forget if Jest requires arrays to have the same order, but if so we could sort them first

```suggestion
      const dumpPaths = dumps.map(d => d.minidumpPath)
      expect(dumpPaths).toEqual([
        join(base, 'report01.dmp'),
        join(base, 'reports', 'report02.dmp'),
        join(base, 'reports', 'report03.dmp')
      ])
```

If not, it'd be good to assert against the length of `dumpPaths`, so we know it doesn't contain "some-other-thing.ps" or any other files